pilot/pkg/model/context.go

```diff
@@ -1338,6 +1338,13 @@ func (node *Proxy) WorkloadEntry() (string, bool) {
 	return node.workloadEntryName, node.workloadEntryAutoCreated
 }
 
+// SupportsEnvoyExtendedJwt indicates that the proxy JWT extension is capable of
+// replacing istio_authn filter.
+func (node *Proxy) SupportsEnvoyExtendedJwt() bool {
```

Comment:
This wouldn't be a breaking change for any user right? Why can't we just cherrypick for all the supported versions instead of maintaining different configs?

Reply:
Yes, it should be fully compatible - it's never 100% but close to it. We can't backport Envoy features, that's against the project policy to backport features and not fixes.